kitchen/modules/weights/src/lib.rs

```diff
@@ -0,0 +1,177 @@
+#![cfg_attr(not(feature = "std"), no_std)]
+
+// Transaction Weight Examples
+// https://crates.parity.io/sr_primitives/weights/index.html
+use support::{ensure, decl_module, decl_storage, dispatch::{Result, WeighData}};
+use runtime_primitives::weights::{ DispatchClass, Weight, ClassifyDispatch, SimpleDispatchInfo };
+
+pub trait Trait: system::Trait {}
+
+decl_storage! {
+	trait Store for Module<T: Trait> as SimpleMap {
+		StoredValue get(stored_value): u32;
```

Comment:
I wonder if the costs of `get()` and `put()` operations scale similarly with requisite transaction weight for other types of storage items. Like a vector or maps.

Reply:
both get and put work in the same was and depend on the encode/decode of the value that you are writing. If you have a vector of 10_000 elements and you want to append 1, you first decode 10_000 elements and then have to encode again 10_001 elements which is exepnsive AF, compared to a normal memory in which you can usually `push()` / `append()` in most languages. This is why we have `storage::Append()` to make this niche use case faster.

Reply:
Added a warning about the importance of characterizing actual performance in the writeup.